### ISSUE TITLE
[figma] Add charts

### ISSUE THREAD

**Issue by oliviertassinari** [mui-org/mui-design-kits#20]
While we don't have any charts in MUI yet, there is a nonnegligible chance that we will get to it.

## Benchmark

- https://antforfigma.com/ant-design-charts-for-figma

**Comment by danilo-leal** [comment]
Yeah, I think we should do it only once we have our own charts released!

**Comment by oliviertassinari** [comment]
@danilo-leal I personally don't resonate with this perspective. We could warn that there are no React implementation available, and recommend a few chart options. I came back to this issue today after seeing this review:

<img width="746" alt="Screenshot 2021-10-29 at 23 12 55" src="https://user-images.githubusercontent.com/3165635/139502364-cdecca3b-ba7d-4ea0-9918-41d7d95abc68.png">

**Comment by danilo-leal** [comment]
But this makes me wonder - and this is a good subject to research - are there people using the Design Kits and _**not**_ using the React library? It's a really strange use case to think about for me. I can imagine people that go to the Kits because they're a more complete set than the MD official ones but even then, what we sell is tailored to be a mirror of our React implementation of the components. And the scenario where designers extract the maximum value out of the Kits is when they have devs using the library, so they share the same language and have no surprises. 

By making a component available first on the Design Kits, and then not having it ready for the React lib, and even going further to recommend other libraries, seems to have the potential to upset people, and it seems to even hurt what we say on the Store: _"What you design is what you get in production"_ - in that case, it won't be, at least not necessarily. 

I guess we can interpret people asking for components, be it through the Design Kits or the library itself, as a general request for a component. And once the component is available on the main medium - React code - it must be available in all others (Design Kits).

**Comment by oliviertassinari** [comment]
> there people using the Design Kits and not using the React library

We would need to survey to know for sure. I won't expect more than 10%

>  not having it ready for the React lib

It could be coming from another library. At least, designers would get some of the pain covered, until we ship our own React chart library, if we ever do.

**Comment by danilo-leal** [comment]
> It's still better than nothing

Initially, it seems it could be but what I'm questioning is exactly this... is it, really?! We'd be adding a component that is not available in our library, linking another library which we don't have control over and no expertise about it which opens the possibility for the design/dev pair using the Figma library disappointed. 

In general, my approach would be adding components to the Design kits only once they're up on the actual library. I wouldn't optimize for the use-case _only using the Kits and not the library_ even though it's possible. I think we provide the greatest value if the designer and developer are synced by using both products. There are already other Figma charts files out there that designers can use. If they're asking us for the charts, I'd suppose they're also asking for the charts to be available on code as well... and if there are many people asking, it's input for prioritization :)

**Comment by oliviertassinari** [comment]
@danilo-leal I can find compelling arguments for both perspectives. Happy either way.

> I'd suppose they're also asking for the charts to be available on code as well...

Could be yes. @adrianmanea feel free to pick the path that resonates the most with you.

**Comment by adrianmanea** [comment]
According to the loud feedback we've got so far, clients seem to value--how close the design kit is compared to the library itself in terms of component props, paddings, etc. I guess the design kit value is measured in development time at the end of the day, with developers spending less time implementing the mockups.

With that being said, what @danilo-leal has commented is aligning with my proposal as well--keep the design kit as close to the library as possible. Having charts would simply be a nice-to-have feature IMHO. Right now, there are other improvements far more valuable which I'm excited to add 🤓.

**Comment by danilo-leal** [comment]
> I guess the design kit value is measured in development time at the end of the day, with developers spending less time implementing the mockups.

Great observation and insight on measuring the Design Kits success :)

**Comment by oliviertassinari** [comment]
Charts: https://mui.com/x/react-charts/

**Comment by oliviertassinari** [comment]
@adrianmanea Should we add this issue in the roadmap: https://github.com/mui/mui-design-kits/projects/1? Seeing https://twitter.com/shpadrian/status/1721460342981534149 suggests that yes

**Comment by adrianmanea** [comment]
Yes, we already have this done in the Q4 zip waiting to be deployed.
